components/map/map.service.ts

```diff
@@ -557,7 +557,9 @@ export class HsMapService {
     visibilityOverrides?: Array<string>
   ): Layer {
     if (removeIfExists && this.layerAlreadyExists(lyr)) {
-      this.removeDuplicate(lyr);
+      if (lyr.get('base') == true) {
```

Comment:
We wont be able to load base layers from compositions this way, no?

Reply:
ok, I guess this.layerAlreadyExists will prevent them not being loaded, but why did they go under base map? I thought that layers are always added on top of the existing. Can't be that base layers are saved last to the permalink on top of the overlays? In that case, that's the cause of the issue, not removingDuplicate.

Reply:
Problem is that layers that are added from app config are added first and only then the permalinks.

Reply:
> We wont be able to load base layers from compositions this way, no?

Shouldnt be problem. Both conditions have to be met - removeIfExists  and layerAlreadyExists